commit 171ff3a34ac166a41a075749517fd3780df121da
Author: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date:   Wed Apr 17 17:26:05 2013 -0400

    Add "sparc" host cpu to configure.ac
    
    Some sparc Debian setups advertise a "sparc" host cpu (rather than
    sparc64).
    
    In all cases, I think it should be safe to add a "sparc" entry to
    userspace RCU configure.ac upstream, e.g.
    
            [sparc], [ARCHTYPE="sparc64"],
    
    in the event someone would launch the build on an environment not
    supporting sparc v9, the build would fail because the 32-bit compiler
    would not be able to generate sparc v9 instructions (unless
    explicitely instructed to do so by the -m32 -Wa,-Av9a flags).
    
    Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>

diff --git a/configure.ac b/configure.ac
index b6f787b..da560a4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -72,6 +72,7 @@ AS_CASE([$host_cpu],
 	[ppc], [ARCHTYPE="ppc"],
 	[s390], [ARCHTYPE="s390"],
 	[s390x], [ARCHTYPE="s390"],
+	[sparc], [ARCHTYPE="sparc64"],
 	[sparc64], [ARCHTYPE="sparc64"],
 	[alpha*], [ARCHTYPE="alpha"],
 	[ia64], [ARCHTYPE="gcc"],
